src/shared/error/schulconnex-validation-error.filter.ts

```diff
@@ -0,0 +1,164 @@
+import { ArgumentsHost, Catch, ExceptionFilter } from '@nestjs/common';
+import { DetailedValidationError } from '../validation/detailed-validation.error.js';
+import { ValidationError } from 'class-validator';
+import { HttpArgumentsHost } from '@nestjs/common/interfaces/index.js';
+import { Response } from 'express';
+
+export type SchulConnexError = {
```

Comment:
It might be useful to export this type to a seperate file

Reply:
✅